Presentation/PresentationStyle.swift

```diff
@@ -121,7 +121,10 @@ public extension PresentationStyle {
                 }
             }
 
-            bag += presented.installDismissButton().onValue {
+            bag += viewController.installDismissButton().onValue {
+                completion(.failure(PresentError.dismissed))
+            }
+            presented.installDismissButton().onValue {
```

Comment:
Sorry Carl, my suggestion was missing the second bag.. 
```suggestion
            bag += presented.installDismissButton().onValue {
```

Reply:
Ah, I should've tested before committing the suggestion too I guess 😄